src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -112,15 +113,20 @@ public class MicrobeEditor : Node, ILoadableGameState
     private List<MeshInstance> placedHexes;
 
     /// <summary>
-    /// The hexes that have been changed by a hovering organelle and need to be reset to old material.
+    ///   The hexes that have been changed by a hovering organelle and need to be reset to old material.
     /// </summary>
     private List<MeshInstance> hexesToResetToOldMaterial;
 
     /// <summary>
-    /// The hexes that have been changed by a hovering organelle and need to be reset to valid material.
+    ///   The hexes that have been changed by a hovering organelle and need to be reset to valid material.
     /// </summary>
     private List<MeshInstance> hexesToResetToValidMaterial;
 
+    /// <summary>
+    ///   The hexes that have been changed by a hovering organelle and need to be reset to island material.
```

Comment:
Reading the code, I don't think this is actually accurate comment. At least it is not the entire picture as the action of marking things as islands, also uses this... See my other comment in the handle islands.